hail_scripts/utils/download_utils.py

```diff
@@ -1,14 +1,32 @@
+from contextlib import contextmanager
 import logging
 import os
 import requests
 import tempfile
-from contextlib import contextmanager
-import hail as hl
 
-from sv_pipeline.utils.common import parse_gs_path_to_bucket
+import hail as hl
+from google.cloud import storage
 
 logger = logging.getLogger(__name__)
 
+def parse_gs_path_to_bucket(gs_path):
```

Comment:
I opted to copy a couple functions that were used from the `sv_pipeline.utils.common` module, which I believe can be deleted after the new SV and GCNV pipelines are merged?